superset-frontend/src/components/DropdownSelectableIcon/index.tsx

```diff
@@ -86,24 +101,40 @@ export default (props: DropDownSelectableProps) => {
             {info}
           </div>
         )}
-        {menuItems.map(m => (
-          <Menu.Item key={m.key}>
-            {m.label}
-            {selectedKeys?.includes(m.key) && (
-              <Icons.Check iconColor={theme.colors.primary.base} />
-            )}
-          </Menu.Item>
-        ))}
+        {menuItems.map(m =>
+          m.children?.length ? (
+            <SubMenu
+              title={m.label}
+              key={m.key}
+              data-test="dropdown-selectable-icon-submenu"
+            >
+              {m.children.map(s => menuItem(s.label, s.key))}
+            </SubMenu>
+          ) : (
+            menuItem(m.label, m.key)
+          ),
+        )}
       </StyledMenu>
     ),
     [info, menuItems],
   );
 
   return (
-    <StyledDropdownButton
-      overlay={overlayMenu}
-      trigger={['click']}
-      icon={icon}
-    />
+    <>
+      <Global
+        styles={css`
+          .ant-dropdown-menu .ant-dropdown-menu-item > .tick-menu-item {
+            float: right;
+            margin-right: 0;
+            font-size: ${theme.typography.sizes.xl}px;
```

Comment:
1. I think we could use Icon's property `iconSize="xl"` instead of `font-size`
2. Maybe we could wrap icon and label in a flex div instead of using `float: right`? That would also allow us not to use `<Global />`

Reply:
Great tip!